setup.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Eventually this should move to pyproject.toml
+#  but setuptools must first gain support for parsing that
+
+import importlib
+import pathlib
+import site
+import sys
+from setuptools import find_packages, setup
+
+here = pathlib.Path(__file__).parent.resolve()
+
+# Add this module to the import path to help centralize metadata
+sys.path.append(str(here) + "/src")
+about = importlib.import_module('decisionengine_modules.about')
+
+# Get the long description from the README file
+long_description = (here / "README.md").read_text(encoding="utf-8")
+
+# pull in runtime requirements
+runtime_require = [
+    "boto >= 2.49.0", "boto3 >= 1.17.10", "google_auth >= 1.16.0, < 2dev",
+    "google-api-python-client >= 1.12.8", "gcs-oauth2-boto-plugin >= 2.7",
+    "packaging >= 20.9", "htcondor >= 8.9.11", "urllib3 >= 1.26.0",
+    "bill-calculator-hep >= 0.1.2",
+    "numpy == 1.19.5; python_version <= '3.6'",
+    "numpy >= 1.19.5; python_version >= '3.7'",
+    "pandas == 1.1.5; python_version <= '3.6'",
+    "pandas >= 1.1.5; python_version >= '3.7'"
+]
+
+# pull in development requirements
+devel_req = [
+    "mock >= 4.0.3", "pytest >= 6.2.2", "pytest-cov >= 2.11.1",
```

Comment:
I believe the `mock` module was a python2 backport of `unittest.mock` from Python3.

Reply:
I replaced the `mock` module with `unittest.mock`.